test/e2e/button/scenarios.js

```diff
@@ -0,0 +1,39 @@
+(function() {
+  'use strict';
+
+  var path = '/test/e2e/button/index.html';
+
+  describe('ons-button', function() {  
+
+    var EC = protractor.ExpectedConditions;
+
+    it('should have an element', function() {
+      browser.get(path);
+      browser.waitForAngular();
+
+      var button = element(by.css('ons-button'));
+
+      //Waits 1000ms for the ons-botton loading
+      browser.wait(EC.presenceOf(button), 1000);
+
+      expect(button.isPresent()).toBeTruthy();
+    });
+    
+    it('should have ng-click or on-click attribute', function() {
+        var button = element(by.css('ons-button'));
+        
+        expect(button.getAttribute('ng-click').isPresent() || button.getAttribute('on-click').isPresent()).toBeTruthy();
```

Comment:
Does this work? I think `isPresent()` returns a promise so it will always be truthy.